File: 920-uncommon-words-from-two-sentences/uncommon-words-from-two-sentences.cpp
class Solution {
public:
    vector<string> uncommonFromSentences(string s1, string s2) {
     map<string,int>m;
     string tmp="";
     for(int i=0;i<s1.size();i++){
        if(s1[i]==' '||i==s1.size()-1){
            if(i==s1.size()-1)
            tmp+=s1[i];
            m[tmp]++;
            tmp="";
        }
        else{
            tmp+=s1[i];
        }
     }
     s1=s2;
      for(int i=0;i<s1.size();i++){
        if(s1[i]==' '||i==s1.size()-1){
            if(i==s1.size()-1)
            tmp+=s1[i];
            m[tmp]++;
            tmp="";
        }
        else{
            tmp+=s1[i];
        }
     }
     vector<string>ans;
     for(auto it:m){
        if(it.second==1)ans.push_back(it.first);
     }
     return ans;
    }
};

File: 920-uncommon-words-from-two-sentences/uncommon-words-from-two-sentences.php

class Solution {

    /**
     * @param String $s1
     * @param String $s2
     * @return String[]
     */
    function uncommonFromSentences($s1, $s2) {
         $map = [];
        $temp = "";
        
        $words1 = explode(' ', $s1);
        foreach ($words1 as $word) {
            if (!isset($map[$word])) {
                $map[$word] = 0;
            }
            $map[$word]++;
        }
        
        $words2 = explode(' ', $s2);
        foreach ($words2 as $word) {
            if (!isset($map[$word])) {
                $map[$word] = 0;
            }
            $map[$word]++;
        }

        $ans = [];
        foreach ($map as $word => $count) {
            if ($count == 1) {
                $ans[] = $word;
            }
        }
        
        return $ans;
    }
}